compatibility-kit/javascript/features/data-tables/data-tables.feature.ndjson

```diff
@@ -1,15 +1,15 @@
 {"source":{"data":"Feature: Data Tables\n  Data Tables can be places underneath a step and will be passed as the last\n  argument to the step definition. They can be used to represent richer data\n  structures, and can also be transformed to other types.\n\n  Scenario: transposed table\n    When the following table is transposed:\n      | a | b |\n      | 1 | 2 |\n    Then it should be:\n      | a | 1 |\n      | b | 2 |\n","uri":"features/data-tables/data-tables.feature","mediaType":"text/x.cucumber.gherkin+plain"}}
-{"gherkinDocument":{"feature":{"tags":[],"location":{"line":1,"column":1},"language":"en","keyword":"Feature","name":"Data Tables","description":"  Data Tables can be places underneath a step and will be passed as the last\n  argument to the step definition. They can be used to represent richer data\n  structures, and can also be transformed to other types.","children":[{"scenario":{"id":"5701b673-f807-4191-ba35-f397ad2c5aac","tags":[],"location":{"line":6,"column":3},"keyword":"Scenario","name":"transposed table","description":"","steps":[{"id":"e465b664-be82-4f8a-ba10-3b71fd84d0b6","location":{"line":7,"column":5},"keyword":"When ","text":"the following table is transposed:","dataTable":{"location":{"line":8,"column":7},"rows":[{"id":"c14298e6-6af0-4002-844e-0a1c5d95dcb7","location":{"line":8,"column":7},"cells":[{"location":{"line":8,"column":9},"value":"a"},{"location":{"line":8,"column":13},"value":"b"}]},{"id":"414c6c5f-6d20-4b9c-9c78-9b01c1c16b66","location":{"line":9,"column":7},"cells":[{"location":{"line":9,"column":9},"value":"1"},{"location":{"line":9,"column":13},"value":"2"}]}]}},{"id":"90c9de7f-f032-4873-84a6-5a68110f1bb6","location":{"line":10,"column":5},"keyword":"Then ","text":"it should be:","dataTable":{"location":{"line":11,"column":7},"rows":[{"id":"23a094ab-ff9d-4653-b524-5d6bfda9dfb8","location":{"line":11,"column":7},"cells":[{"location":{"line":11,"column":9},"value":"a"},{"location":{"line":11,"column":13},"value":"1"}]},{"id":"99dec49b-9cbb-410d-b52a-2be096d1bc10","location":{"line":12,"column":7},"cells":[{"location":{"line":12,"column":9},"value":"b"},{"location":{"line":12,"column":13},"value":"2"}]}]}}],"examples":[]}}]},"comments":[],"uri":"features/data-tables/data-tables.feature"}}
-{"pickle":{"id":"41935011-5dc1-4152-aaff-7af50edaf4dc","uri":"features/data-tables/data-tables.feature","astNodeIds":["5701b673-f807-4191-ba35-f397ad2c5aac"],"tags":[],"name":"transposed table","language":"en","steps":[{"id":"424fff5d-36db-4fcf-9e03-3676669e2931","text":"the following table is transposed:","argument":{"dataTable":{"rows":[{"cells":[{"value":"a"},{"value":"b"}]},{"cells":[{"value":"1"},{"value":"2"}]}]}},"astNodeIds":["e465b664-be82-4f8a-ba10-3b71fd84d0b6"]},{"id":"5f5f5a16-74ad-4298-ad07-cac1f2989498","text":"it should be:","argument":{"dataTable":{"rows":[{"cells":[{"value":"a"},{"value":"1"}]},{"cells":[{"value":"b"},{"value":"2"}]}]}},"astNodeIds":["90c9de7f-f032-4873-84a6-5a68110f1bb6"]}]}}
```

Comment:
Side note: what do we think about using predictable ids and timing in `fake-cucumber` for this? I know we still wouldn't expect implementations to match exactly, but would at least stop every line in every fixture always changing.

Reply:
Yes, I think that's a good idea.

Reply:
Okay - I'll make a separate PR for that once this lands.

Reply:
https://github.com/cucumber/common/pull/1639